test/src/test/resources/fixtures/__typename-is-passed-on-queries-using-dynamic-resolved-services.yml

```diff
@@ -0,0 +1,95 @@
+name: __typename is passed on queries using dynamic resolved services
```

Comment:
dynamic service resolution wasn't affected because it checks only top level fields